docker_files/etc/confd/templates/config.json.tmpl

```diff
@@ -1,120 +1,85 @@
-		"max": {{getv "/lisk/db/max" "95"}},
-		"poolIdleTimeout": {{getv "/lisk/db/poolidletimeout" "30000"}},
-		"reapIntervalMillis": {{getv "/lisk/db/reapintervalmillis" "1000"}},
-		"logEvents": ["error"],
-		"logFileName": "{{getv "/lisk/db/logfilename" "logs/lisk_db.log"}}"
+		"host": "{{getv "/lisk/db/host" "null"}}",
+		"port": {{getv "/lisk/db/port" "null"}},
+		"database": "{{getv "/lisk/db/database" "null"}}",
+		"user": "{{getv "/lisk/db/user" "null"}}",
+		"password": "{{getv "/lisk/db/password" "null"}}",
+		"min": {{getv "/lisk/db/min" "null"}},
+		"max": {{getv "/lisk/db/max" "null"}},
+		"poolIdleTimeout": {{getv "/lisk/db/poolidletimeout" "null"}},
+		"reapIntervalMillis": {{getv "/lisk/db/reapintervalmillis" "null"}},
+		"logEvents": {{getv "/lisk/db/logevents" "null"}},
+		"logFileName": "{{getv "/lisk/db/logfilename" "null"}}"
 	},
 	"redis": {
-		"host": "{{getv "/lisk/redis/host" "127.0.0.1"}}",
-		"port": {{getv "/lisk/redis/port" "6380"}},
-		"db": {{getv "/lisk/redis/db" "0"}},
+		"host": "{{getv "/lisk/redis/host" "null"}}",
+		"port": {{getv "/lisk/redis/port" "null"}},
+		"db": {{getv "/lisk/redis/db" "null"}},
 		"password": {{getv "/lisk/redis/password" "null"}}
 	},
 	"api": {
-		"enabled": {{getv "/lisk/api/enabled" "true"}},
+		"enabled": {{getv "/lisk/api/enabled" "null"}},
 		"access": {
-			"public": {{getv "/lisk/api/access/public" "false"}},
+			"public": {{getv "/lisk/api/access/public" "null"}},
 			{{$whiteListEntries := getvs "/lisk/api/access/whitelist/*"}}"whiteList": [{{range $index, $element := $whiteListEntries}}{{if $index}}, {{end}}"{{$element}}"{{end}}]
 		},
 		"options": {
 			"limits": {
-				"max": {{getv "/lisk/api/options/limits/max" "0"}},
-				"delayMs": {{getv "/lisk/api/options/limits/delayms" "0"}},
-				"delayAfter": {{getv "/lisk/api/options/limits/delayafter" "0"}},
-				"windowMs": {{getv "/lisk/api/options/limits/windowms" "60000"}}
+				"max": {{getv "/lisk/api/options/limits/max" "null"}},
+				"delayMs": {{getv "/lisk/api/options/limits/delayms" "null"}},
+				"delayAfter": {{getv "/lisk/api/options/limits/delayafter" "null"}},
+				"windowMs": {{getv "/lisk/api/options/limits/windowms" "null"}}
 			},
 			"cors": {
-				"origin": "{{getv "/lisk/api/options/cors/origin" "*"}}",
-				"methods": ["GET", "POST", "PUT"]
+				"origin": "{{getv "/lisk/api/options/cors/origin" "null"}}",
+			        {{$corsMethods := getvs "/lisk/api/options/cors/methods/*"}}"methods": [{{range $index, $element := $corsMethods}}{{if $index}}, {{end}}"{{$element}}"{{end}}]
+			}
+		},
+		"ssl": {
+			"enabled": {{getv "/lisk/ssl/enabled" "null"}},
+			"options": {
+				"port": {{getv "/lisk/ssl/options/port" "null"}},
+				"address": "{{getv "/lisk/ssl/options/address" "null"}}",
+				"key": "{{getv "/lisk/ssl/options/key" "null"}}",
+				"cert": "{{getv "/lisk/ssl/options/cert" "null"}}"
 			}
 		}
 	},
 	"peers": {
-		"enabled": {{getv "/lisk/peers/enabled" "true"}},
-		"list": [
```

Comment:
I would prefer to have the opportunity to choose my peers. I don't see the point to force Docker users to use Lisk Official peers.